src/components/layout/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Container, makeStyles } from '@material-ui/core';
+
+type LayoutProps = {
+    children: JSX.Element[] | JSX.Element
+};
+
+const useStyles = makeStyles(() => ({
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        width: "350px",
+        height: "600px",
+        border: "1px solid #1976d2",
+        borderRadius: "8px",
+    },
+}));
+
+export default function Layout ({ children }: LayoutProps) {
+    const classes = useStyles();
+    return (
+        <Container className={classes.container}>
+            {children}
+        </Container>
+    );
+};
```

Comment:
I'd like to call it an `abstract` component, but I'm not sure this kind of component has its name.
This component will be reused per every `page` component.